product/infrastructure/fake/searchservice_test.go

```diff
@@ -258,10 +270,28 @@ func TestSearchService_SearchBy(t *testing.T) {
 			s := new(fake.SearchService).Inject(
 				new(fake.ProductService),
 				&struct {
-					LiveSearchJSON         string `inject:"config:commerce.product.fakeservice.jsonTestDataLiveSearch,optional"`
-					CategoryFacetItemsJSON string `inject:"config:commerce.product.fakeservice.jsonTestDataCategoryFacetItems,optional"`
+					LiveSearchJSON         string       `inject:"config:commerce.product.fakeservice.jsonTestDataLiveSearch,optional"`
+					CategoryFacetItemsJSON string       `inject:"config:commerce.product.fakeservice.jsonTestDataCategoryFacetItems,optional"`
+					SortConfig             config.Slice `inject:"config:commerce.product.fakeservice.sorting"`
 				}{
 					LiveSearchJSON: tt.fields.liveSearchJSON,
+					SortConfig: config.Slice{
+						testSortConfig{
+							Key:   "camera",
+							Label: "camera",
+							Asc:   "camera",
+						},
+						testSortConfig{
+							Key:   "size",
+							Label: "size",
+							Desc:  "size",
+						},
+						testSortConfig{
+							Key:   "no-direction",
+							Label: "no-direction",
+							Asc:   "no-direction",
```

Comment:
can we have an example with both asc/desc set?

Reply:
added